test/baselines/generated-reference/algorithm-replacements.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<head><meta charset="utf-8"></head><body><div id="shortcuts-help">
+<head><meta name="viewport" content="width=device-width, initial-scale=1"><meta charset="utf-8"></head><body><div id="shortcuts-help">
```

Comment:
Kind of unfortunate this ends up being the second element but it's still close enough to the beginning of the file so it's probably fine.